src/Pages/Home.js

```diff
@@ -71,9 +71,37 @@ function Home(){
                         <img src= {require('../Assets/Profile_placeholder.jpeg')} 
                         alt=""/>
                     </div>
-                    <button className="Font_style_2">Resume</button>
+                    <button 
+                    className="Font_style_2 button_style_1">
+                        Resume
+                    </button>
                 </div>
             </div> 
+            {/* Footer Section */}
+            <div className="Footer_section">
+                <div className="Footer_container_1">
+                    <p className="Font_style_3">What's Next?</p>
+                    <h2 className="Font_style_4">GET IN TOUCH</h2>
+                </div>
+                <div className="Footer_container_2">
+                    <p className="Font_style_3">
+                        Feel free to get in touch with me to discuss any potential 
+                        projects or collaborations. Whether you want to explore a new 
+                        idea or just say hello, I'm always open to new opportunities.
+                    </p>
+                    <button 
+                    className="Font_style_3 button_style_1">
+                        Get in <br/> touch
+                    </button>
+                </div>
+                <div className="Footer_container_3">
+                    <img src= {require('../Assets/code-branch.png')} 
+                    alt=""/>
```

Comment:
doesn’t look very clean to have the “alt” tag on it’s own line. Unless the src is very long, I would keep the src and alt tag on the same line to keep things condensed and clean. 
I recommend looking at the other img elements you have across the app and use your best judgment on whether they need 1 or more lines.